week-1/errors/3.js

```diff
@@ -1,7 +1,9 @@
-const cardNumber = 4533787178994213;
+const cardNumber = "4533787178994213";
 const last4Digits = cardNumber.slice(-4);
 
 // The last4Digits variable should store the last 4 digits of cardNumber
 // However, the code isn't working
 // Make and explain a prediction about why the code won't work
 // Then try updating the expression last4Digits is assigned to, in order to get the correct value
+
+Slice() method is a part of a string, not a number. CardNumber becomes a string when enclosed in single quotations. 
```

Comment:
Brilliant explanation, but just remember .slice is not part of a string. Instead, it extracts a specified part of a string.